_id/1/index.json

```
@@ -0,0 +1,34 @@
+{
+	"pathwayTitle": "Kubernetes Bootcamp",
+	"title": "Modul 1 - Membuat sebuah klaster Kubernetes",
+	"description": "Membuat sebuah klaster Kubernetes",
+	"backend": {
+		"port": 30000,
+		"imageid": "minikube"
+	},
+	"environment": {
+		"uilayout": "terminal",
+		"hideintro": false,
+		"hidefinish": false
+	},
+	"details": {
+		"steps": [{
+			"title": "Klaster dimulai dan berjalan",
+			"text": "step1.md"
+		}, {
+			"title": "Versi klaster",
+			"text": "step2.md"
+		}, {
+			"title": "Detail klaster",
+			"text": "step3.md"
```

how about opening the steps with active verbs instead? seems more intuitive in Indonesian language for tutorial
```suggestion
			"title": "Memulai dan menjanlankan Klaster",
			"text": "step1.md"
		}, {
			"title": "Melihat versi klaster",
			"text": "step2.md"
		}, {
			"title": "Melihat detail klaster",
			"text": "step3.md"
```

---

Yup, more obvious